raiden-dapp/src/locales/en.json

```diff
@@ -468,6 +468,18 @@
         "File a new issue on https://github.com/raiden-network/light-client",
         "Upload logs to the issue"
       ]
+    },
+    "PFS_NO_ROUTES_BETWEEN_NODES": {
+      "title": "No route between you and your target",
+      "problem": "The Pathfinding Service was not able to find a route between you and the target you want to transfer tokens to.",
```

Comment:
`find a usable route`; there may be a route but it is unusable for more than one reason: not enough capacity on our side or one of the mediating channels; some peer not online; etc